src/LibraryManager.Vsix/Commands/InstallLibraryCommand.cs

```diff
@@ -122,8 +122,10 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
                 // This will remove the dialog from alt-tab list and will not allow the user to switch the dialog box to the background 
                 windowInteropHelper.Owner = new IntPtr(hwnd);
 
-                dialog.ShowDialog();
+                    dialog.ShowDialog();
```

Comment:
Formatting of this line and the one below is off

Reply:
Did not see this on time. I guess it got messed up during the merge